modules/channel-vonage/src/actions/storeFileLocally.js

```diff
@@ -48,15 +49,33 @@ const storeFileLocally = async () => {
   const fileUrl = event.payload[prop]
 
   try {
-    const resp = await axios.get(fileUrl, { responseType: 'arraybuffer' })
+    const TIMEOUT = 5000 // timeout of 5 seconds
```

Comment:
as a matter of consistency around the codebase 
```suggestion
    const TIMEOUT = ms('5s')
```